packages/cdk-environment-resources/lib/index.ts

```diff
@@ -20,6 +23,40 @@ export class CdkEnvironmentResources extends cdk.Construct {
 
         const vpc = ec2.Vpc.fromLookup(this, "VPC", { vpcId: props.vpcId });
 
+        const bucket = new s3.Bucket(this, 'Bucket', {
+            bucketName: `deploy-dsicollection.${props.stage}.ecs`,
+            publicReadAccess: true,
+            versioned: true,
+        });
+
+        const originAccessIdentity = new cloudfront.OriginAccessIdentity(this, 'OriginAcessIdentity', {
```

Comment:
You actually don't need the origin access identity and the bucket policy. That is just granting permissions for CloudFront to access the files in your bucket but since these will in theory will only be read by CodeDeploy and CloudFormation this won't be necessary.